fr.henri2h.minestrix.json

```diff
@@ -0,0 +1,112 @@
+{
+    "app-id": "fr.henri2h.minestrix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "minestrix",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.gnome.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "only_arches": [
+                "x86_64",
+                "aarch64"
+            ],
+            "buildsystem": "meson",
+            "config-opts": [
+                "--buildtype=release",
+                "--default-library=shared"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.tar.gz",
+                    "sha256": "f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2"
+                }
+            ]
+        },
+        {
+            "name": "olm",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.matrix.org/matrix-org/olm.git",
+                    "tag": "3.2.14"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
```

Comment:
it is in the runtime, why do yo need to build it?

Reply:
ok, didn't new, going to remove it.